Backend_NodeJS/src/api/controllers/products/edit/edit-product.js

```diff
@@ -0,0 +1,29 @@
+const { Product } = require("../../../../models/index.js");
+
+module.exports = async (req, res) => {
+  try {
+    const data = req.body;
+    const { _id } = data;
+    delete data._id;
+    console.log(data);
+
+    // Use the { new: true } option to return the updated document
+    let product = await Product.findOneAndUpdate(
+      { _id },
+      { ...data },
```

Comment:
Here in edit product API Instead of updating the product directly without checking what data is being passed by the user we can validate the data that is being passed and if the correct data is passed then only  we can use that data to update product